src/main/java/com/rybka/Constant.java

```diff
@@ -0,0 +1,6 @@
+package com.rybka;
+
+public final class Constant {
+    public static final String URL = "https://prime.exchangerate-api.com/v5/%s/latest/%s";
```

Comment:
URL isn't obvious. CURRENCY_API_URL better as for me

Reply:
The same for API_KEY. 
Or you can create CurrencyAPIConstanst and keep API_URL, API_KEY there.

Reply:
fixed